src/types.ts

```diff
@@ -95,18 +96,22 @@ export interface EventBus extends ErrorBus {
   off(name: string, callback: (event: unknown) => void): this
 }
 
-export interface ILiveConnect {
-  ready: boolean
-  push: (event: unknown) => void
-  fire: () => void
+export interface PublicLiveConnect {
+  push: (...event: unknown[]) => void
+}
```

Comment:
Would it make sense to make the resolve function and config part of the public interface?

Reply:
Resolve yes, I would say config is probably not something users should use